wandb/sdk/wandb_settings.py

```diff
@@ -556,6 +559,18 @@ def apply_init(self, args):
             group="run_group",
             dir="root_dir",
         )
+
+        if self["_project_override"]:
+            current_project = args.get("project") or self["project"]
+            if current_project:
+                logger.info(
```

Comment:
This log doesn't show up, but I'm sure I'm hitting this line -- is there something subtle I need to do with retrieving an early logger?